README.md

```diff
@@ -1,8 +1,13 @@
 # Decidim::Vocdoni
 
+[![[CI] Lint](https://github.com/decidim-vocdoni/decidim-module-vocdoni/actions/workflows/lint.yml/badge.svg)](https://github.com/decidim-vocdoni/decidim-module-vocdoni/actions/workflows/lint.yml)
+[![[CI] Tests](https://github.com/decidim-vocdoni/decidim-module-vocdoni/actions/workflows/test.yml/badge.svg)](https://github.com/decidim-vocdoni/decidim-module-vocdoni/actions/workflows/test.yml)
+[![Maintainability](https://api.codeclimate.com/v1/badges/126b8ece66b8292802f3/maintainability)](https://codeclimate.com/github/decidim-vocdoni/decidim-module-vocdoni/maintainability)
+
+
 :warning: This module is under development and is not ready to be used in production.
 
-An elections component for decidim's participatory spaces based on the [Vocdoni][vocdoni-app-url].
+An elections component for decidim's participatory spaces based on the [Vocdoni][https://vocdoni.app].
```

Comment:
As we're moving from the Markdown reference links, we need to change the format too: 

![image](https://user-images.githubusercontent.com/717367/217754454-66c234b8-08fb-4938-8813-0a268e996c1e.png)


```suggestion
An elections component for decidim's participatory spaces based on the [Vocdoni](https://vocdoni.app).
```

(I'm a fan of reference links, but I can leave with the links duplicated too.)

Reply:
reference links tends to confuse me, but i can live with them too